src/batch-client/index.ts

```diff
@@ -956,6 +956,11 @@ export class ZimbraBatchClient {
 			);
 		});
 
+	public getImportExternalAccountStatus = () =>
+		this.jsonRequest({
+			name: 'GetImportStatus'
```

Comment:
usually most of the function and schema names are aligned with api name, to reduce confusion

Reply:
Yeah, Right. I was trying to make method name self-explanatory. 😄